pandas/tests/arithmetic/test_numeric.py

```diff
@@ -66,7 +66,12 @@ def test_df_numeric_cmp_dt64_raises(self):
         ts = pd.Timestamp.now()
         df = pd.DataFrame({"x": range(5)})
 
-        msg = "Invalid comparison between dtype=int64 and Timestamp"
+        msg = "|".join(
+            [
+                "Invalid comparison between dtype=int64 and Timestamp",
+                "'[<>]' not supported between instances of 'Timestamp' and 'int'",
+            ]
+        )
```

Comment:
can we test explicitly with one message and then the other instead? or even better can we make the error messages consistent?

Reply:
yep, turns out we only need the new one, updated.